src/commonmark/en/content/developer/web-api.md

```diff
@@ -2124,81 +2124,67 @@ following payload to change the style:
-unit, etc.), action is *CREATE, UPDATE, DELETE* and *id* is the id of the
-type being handled. The event also contains a payload, for *CREATE* and
-*DELETE* this is the full serialized version of the object, for *UPDATE*
-its a patch containing the updates that are happening on that object.
-
-## Kafka integration
-
-<!--DHIS2-SECTION-ID:webapi_kafka_integration-->
-
-Kafka integration was added in 2.30 and is currently supported for doing
-bulk imports of tracker data (tracked entities, enrollments, events), to
-integrate with kafka you need to first install and configure Kafka (see
-<https://docs.confluent.io/current/installation/installing_cp/index.html>
-for more information), then update your "dhis.conf" with at least the
-"*kafka.bootstrap-servers*" key, an example can be seen below (only
-*bootstrap-servers* is required, the rest is just to show the defaults):
-
-    kafka.bootstrap-servers = localhost:9092
-    kafka.client-id = dhis2
-    kafka.retries = 10
-    kafka.max-poll-records = 1000
-
-After adding this, you will need to create the topics that you are
-interested in using, below we will show how that is done for tracker
-data (the only currently supported topics).
-
-### Kafka tracker integration
-
-For doing bulk imports of tracker data, 3 new endpoints has been
-created:
-
-    /api/trackedEntityInstances/queue (topic: bulk-tracked-entities)
-    /api/enrollments/queue (topic: bulk-enrollments)
-    /api/events/queue (topic: bulk-events)
-
-You can use these endpoints exactly as you would with the normal tracker
-endpoint, the only difference is that they are all asynchronous, and you
-will need to get the import summaries using normal task summary
-approach. Internally there is a scheduled system job called "Kafka Job"
-which runs every second to check for new objects waiting in Kafka.
-
-The topics will be created by DHIS2 if they don't exist, but be aware
-that they will then have their basic default settings applied (1
-replication factor, and 1 partition only).
+## ActiveMQ Artemis / AMQP 1.0 integration
+
+<!--DHIS2-SECTION-ID:webapi_amqp_integration-->
+
+By default DHIS2 will start up an embedded instance of ActiveMQ Artemis when the
+instance is booting up. For most use-cases you do not need to configure anything
+to make use of this, but if you infrastructure have an existing AMQP 1.0 compliant
+service you want to use, you can change the defaults in your *dhis.conf* file using
+the keys in the table down below.
+
+<table>
+  <caption>
+    AMQP Configuration Keys
+  </caption>
+  <colgroup>
+    <col style="width: 15%" />
+    <col style="width: 30%" />
+    <col style="width: 55%" />
+  </colgroup>
+  <thead>
+    <tr class="header">
+      <th>Key</th>
+      <th>Value (default first)</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr class="odd">
+      <td>amqp.mode</td>
+      <td><code>EMBEDDED</code> | <code>NATIVE</code></td>
+      <td>Host to bind to.</td>
```

Comment:
Wrong text here - should be an explanation of the mode.

Reply:
Should be ok now